src/components/mc/mc_Node.bs

```diff
@@ -24,6 +24,7 @@ function init()
   m.notifyChanges = true
   m.pendingChanges = []
   m.children = []
+  m.findNodeById = ""
```

Comment:
this is not the way to do this. we do not want a side effect of a find function looking like state.